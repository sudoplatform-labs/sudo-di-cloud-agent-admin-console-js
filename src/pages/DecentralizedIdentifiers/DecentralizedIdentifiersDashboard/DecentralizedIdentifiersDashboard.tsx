import React from 'react';
import { PageHeading, Page } from '../../../components/NavLayout';
import { DecentralizedIdentifiersCard } from '../DecentralizedIdentifiersCard';

export const DecentralizedIdentifiersDashboard: React.FC = () => {
  return (
    <Page>
      <PageHeading>Decentralized Identifiers</PageHeading>
      <DecentralizedIdentifiersCard />
    </Page>
  );
};
