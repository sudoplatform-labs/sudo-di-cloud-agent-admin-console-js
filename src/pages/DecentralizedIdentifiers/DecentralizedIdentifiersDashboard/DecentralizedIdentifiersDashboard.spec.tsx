import { shallow } from 'enzyme';
import React from 'react';
import { DecentralizedIdentifiersDashboard } from './DecentralizedIdentifiersDashboard';

describe('pages/DecentralizedIdentifiersDashboard', () => {
  it('Should render', () => {
    const wrapper = shallow(<DecentralizedIdentifiersDashboard />);
    expect(wrapper).toMatchSnapshot();
  });
});
