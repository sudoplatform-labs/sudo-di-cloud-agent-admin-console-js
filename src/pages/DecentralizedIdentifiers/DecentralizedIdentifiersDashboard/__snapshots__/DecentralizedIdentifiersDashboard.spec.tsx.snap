// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages/DecentralizedIdentifiersDashboard Should render 1`] = `
<styled.div>
  <styled.h1>
    Decentralized Identifiers
  </styled.h1>
  <DecentralizedIdentifiersCard />
</styled.div>
`;
